scripts/scanResourcesComponents/cloudFrontCachePolicy.ts

```diff
@@ -0,0 +1,35 @@
+#!npx tsx
```

Comment:
What are these at the top of the files?
I'd love to stick with yarn if possible since that's what all the apps use.
But I also don't really understand these lines so if it's something else, ignore me :)

Reply:
this shebang is telling these files how to execute if run directly, we should be able to add yarn.